Misc/NEWS.d/next/Library/2020-05-05-06-05-24.bpo-40507.ggjoGl.rst

```diff
@@ -0,0 +1 @@
+The os.exec* family of functions now includes the filename when raising a FileNotFound error.
```

Comment:
I'd re-word this slightly to explicitly mention that the `filename` attribute is set on the exception object.

Reply:
@taleinat I've updated the wording as suggested